app/models/case/ico/sar.rb

```diff
@@ -1,8 +1,17 @@
 class Case::ICO::SAR < Case::ICO::Base
 
+  has_paper_trail only: [
```

Comment:
Won't this override the `has_paper_trail` in `Case::ICO::Base`? That one contains fields we want to track like `ico_reference_number` which should be in here, or better yet just use that?